state-transition/core/core_test.go

```diff
@@ -227,3 +227,10 @@ func buildNextBlock(
 		Body:          nextBlkBody,
 	}
 }
+
+// We verify DepositRoot in Eth1Data, but we don't really update
+// the store once we store deposits (it's not StateProcessor responsibility).
+// So we just keep using the dummyEth1Data most of the time.
+var dummyEth1Data = &types.Eth1Data{
```

Comment:
no longer needed

Reply:
addressed in https://github.com/berachain/beacon-kit/pull/2304